static/sb-styles/preview.css

```diff
@@ -10,7 +10,7 @@ html {
     padding: 0.8rem;
 }
 .sbdocs.sbdocs-wrapper {
-    padding: 3.2rem 20px;
+    padding: 0 20px 3.2rem 20px;
```

Comment:
Do we use `px` instead of `rem` for the horizontal padding here so that it's fixed? 

This isn't specific to typography rems but got me thinking, would we want to provide a way for devs to use the `px` version of our tokens?

Reply:
Good question! I had to override the top padding so it didn't get overlapped. But it had pixels before, so I kept them there. I will switch them to rems!